src/upload_all_java_tools.sh

```diff
@@ -48,7 +48,7 @@ bazel_version=$(bazel info release | cut -d' ' -f2)
 
 # Passing the same commit_hash and timestamp to all targets to mark all the artifacts
 # uploaded on GCS with the same identifier.
-for java_version in 11 14; do
+for java_version in 11 14 15; do
```

Comment:
Can you rebase to master and leave only 11 here. With a recent commit to master I realised we can have only a single release of java_toolchains, no need to multiply here.

Reply:
Done.